supporting_content/columns/pricingcategory.md

```diff
@@ -14,68 +14,69 @@ Current column mappings found in available data sets:
 
 Current values observed in billing data for various scenarios:
 
-| Provider  | Data set                    | Provider value     | PricingCategory  | PricingSubcategory |
-| --------- | --------------------------- | ------------------ | ---------------- | ------------------ |
-| AWS       | CUR (PurchaseOption)        | On-Demand          | On-Demand        | (depends on usage) |
-| AWS       | CUR (PurchaseOption)        | Reserved Instances | Commitment-Based | Committed Usage    |
-| AWS       | CUR (PurchaseOption)        | Spot Instances     | Dynamic          | Spot               |
-| AWS       | CUR (PurchaseOption)        | Dedicated Hosts    | On-Demand        | (depends on usage) |
-| Microsoft | Cost Details (PricingModel) | OnDemand           | On-Demand        |
-| Microsoft | Cost Details (PricingModel) | SavingsPlan        | Commitment-Based | Committed Spend    |
-| Microsoft | Cost Details (PricingModel) | Reservation        | Commitment-Based | Committed Usage    |
-| Microsoft | Cost Details (PricingModel) | Spot               | Reservation      |
+| Provider  | Data set                    | Provider value     | PricingCategory | PricingSubcategory |
+| --------- | --------------------------- | ------------------ | --------------- | ------------------ |
+| AWS       | CUR (PurchaseOption)        | On-Demand          | Standard        | (depends on usage) |
+| AWS       | CUR (PurchaseOption)        | Reserved Instances | Committed       | Committed Usage    |
+| AWS       | CUR (PurchaseOption)        | Spot Instances     | Dynamic         | Spot               |
+| AWS       | CUR (PurchaseOption)        | Dedicated Hosts    | Standard        | (depends on usage) |
+| Microsoft | Cost Details (PricingModel) | OnDemand           | Standard        |                    |
+| Microsoft | Cost Details (PricingModel) | SavingsPlan        | Committed       | Committed Spend    |
+| Microsoft | Cost Details (PricingModel) | Reservation        | Committed       | Committed Usage    |
+| Microsoft | Cost Details (PricingModel) | Spot               | Dynamic         |                    |
 
 ## Documentation
 
-- AWS: https://docs.aws.amazon.com/cur/latest/userguide/product-columns.html
-- GCP
-- Microsoft: https://learn.microsoft.com/azure/cost-management-billing/automate/understand-usage-details-fields
+* AWS: https://docs.aws.amazon.com/cur/latest/userguide/product-columns.html
+* GCP
+* Microsoft: https://learn.microsoft.com/azure/cost-management-billing/automate/understand-usage-details-fields
 
 ## Discussion Topics
 
-- Goal of the column is to enable practitioners to identify charges that have reduced prices vs. not.
```

Comment:
No changes to this section. I just standardized to use `*` instead of `-` for bullets.